app/models/subnet.rb

```diff
@@ -187,7 +187,14 @@ def self.import(proxy)
     ProxyAPI::DHCP.new(:url => proxy.url).subnets.map do |s|
       # do not import existing networks.
       attrs = { :network => s["network"], :mask => s["netmask"] }
+
       next if first(:conditions => attrs)
+      if s["options"]
+        attrs.merge!({:gateway => s["options"]["gateway"],
```

Comment:
nm, I see that `options` hash has been added in that PR. I'd change this end to iterate over all keys available in `options` and append them to `attrs`.